pkg/cmd/convert.go

```diff
@@ -62,20 +69,44 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
 	for _, file := range files {
```

Comment:
@surajssd I didn't edit this part of code in this PR, let's send this a separate one?